src/Psalm/Checker/Statements/Expression/CallChecker.php

```diff
@@ -318,11 +318,11 @@ public static function analyzeFunctionCall(
                     $stmt->args,
                     $method_id,
                     $function_params,
+                    $code_location,
```

Comment:
No – I want the docblock to be totally unchanged – just remove the type hinting in on the parameter types:

```php
/**
     * @param   StatementsChecker                       $statements_checker
     * @param   array<int, PhpParser\Node\Arg>          $args
     * @param   string|null                             $method_id
     * @param   array<int,FunctionLikeParameter>        $function_params
     * @param   FunctionLikeStorage                     $function_storage
     * @param   ClassLikeStorage                        $class_storage
     * @param   array<string, Type\Union>|null          $generic_params
     * @param   CodeLocation                            $code_location
     * @param   bool                                    $check_variables
     * @return  false|null
     */
    protected static function checkFunctionArgumentsMatch(
        StatementsChecker $statements_checker,
        array $args,
        $method_id,
        array $function_params,
        $function_storage,
        $class_storage,
        &$generic_params,
        CodeLocation $code_location,
        $check_variables
    ) {
```